src/styles/partials/components/_images.scss

```diff
@@ -0,0 +1,42 @@
+.dg_image {
+  display: block;
+  position: relative;
+  img {
+    display: block;
+    object-fit: fill;
+    width: 100%;
+  }
+}
+
+.left {
```

Comment:
`.left` and `.right` should next below `.dg_image`

Reply:
done

Reply:
sorry, nested in `.dg_image`

```scss
.dg_image {
  display: block;
  ...

  .left {
     something-goes: here;
  }
}
```

Reply:
same with .right